server/monitor.go

```diff
@@ -2377,22 +2377,25 @@ func (s *Server) accountInfo(accName string) (*AccountInfo, error) {
 
 // JSzOptions are options passed to Jsz
 type JSzOptions struct {
-	Account    string `json:"account,omitempty"`
-	Accounts   bool   `json:"accounts,omitempty"`
-	Streams    bool   `json:"streams,omitempty"`
-	Consumer   bool   `json:"consumer,omitempty"`
-	Config     bool   `json:"config,omitempty"`
-	LeaderOnly bool   `json:"leader_only,omitempty"`
-	Offset     int    `json:"offset,omitempty"`
-	Limit      int    `json:"limit,omitempty"`
+	Account     string `json:"account,omitempty"`
+	Accounts    bool   `json:"accounts,omitempty"`
+	Streams     bool   `json:"streams,omitempty"`
+	Consumer    bool   `json:"consumer,omitempty"`
+	Config      bool   `json:"config,omitempty"`
+	LeaderOnly  bool   `json:"leader_only,omitempty"`
+	Offset      int    `json:"offset,omitempty"`
+	Limit       int    `json:"limit,omitempty"`
+	Replication bool   `json:"replication,omitempty"`
```

Comment:
We can probably just assume giving replication details are needed if stream details were requested I think.

This looks a lot nicer and smaller than before well done.

Reply:
Sounds good, I was just flagging it because it might be a performance issue, if not, that's easier! Updated.